src/types.ts

```diff
@@ -45,17 +45,30 @@ export type FormField = {
 // <SubmissionPublicKey>;<Base64Nonce>:<Base64EncryptedData>
 export type EncryptedContent = string
 
+// Records containing a map of field IDs to URLs where encrypted
+// attachments can be downloaded.
+export type EncryptedAttachmentRecords = Record<string, string>
+
 export interface DecryptParams {
   encryptedContent: EncryptedContent
   version: number
   verifiedContent?: EncryptedContent
+  attachmentDownloadUrls?: EncryptedAttachmentRecords
 }
 
 export type DecryptedContent = {
   responses: FormField[]
   verified?: Record<string, any>
 }
 
+export type DecryptedFile = {
+  filename: string
+  content: Uint8Array | null
```

Comment:
is it actually okay for the `content` to be `null`?

Reply:
Yeah I was just thinking of using whatever decryptFile returned, but I will just return a `null` for the whole submission in this case.